homeassistant/components/hassio/services.yaml

```diff
@@ -1,37 +1,84 @@
 addon_install:
   description: Install a HassIO docker addon.
   fields:
-    addon: {description: Name of addon., example: smb_config}
-    version: {description: Optional or it will be use the latest version., example: '0.2'}
+    addon:
+      description: Name of addon.
+      example: smb_config
```

Comment:
While the example is valid, we do not have an add-on named that way.
`core_ssh` might be a better example to use since that is an add-on that a lot of people use and can relate to.

Reply:
implemented